src/main/Extensions/ExtensionLoader.ts

```diff
@@ -2,6 +2,7 @@ import type { Dependencies } from "@Core/Dependencies";
 import type { DependencyRegistry } from "@Core/DependencyRegistry";
 import { AppearanceSwitcherModule } from "./AppearanceSwitcher";
 import { ApplicationSearchModule } from "./ApplicationSearch";
+import { Base64ConversionModule } from "./Base64Conversion/Base64ConversionModule";
```

Comment:
...then you can also simplify the import here:

```suggestion
import { Base64ConversionModule } from "./Base64Conversion";
```